src/common/schemas/LineProtocolError.yml

```diff
@@ -3,19 +3,20 @@
       $ref: "./LineProtocolErrorCode.yml"
     message:
       readOnly: true
-      description: Message is a human-readable message.
+      description: Human-readable message.
       type: string
     op:
       readOnly: true
-      description: Op describes the logical code operation during error. Useful for debugging.
+      description: Describes the logical code operation when the error occurred. Useful for debugging.
       type: string
     err:
       readOnly: true
-      description: Err is a stack of errors that occurred during processing of the request. Useful for debugging.
+      description: Stack of errors that occurred during processing of the request. Useful for debugging.
       type: string
     line:
       readOnly: true
-      description: First line within sent body containing malformed data
+      description: First line in the request body that contains malformed data.
       type: integer
       format: int32
-  required: [code, message, op, err]
+  required:
+    - code
```

Comment:
@glinton @lesam Can you verify this change is OK? Just calling it out so I don't break a client.